fast/stages/2-security/variables.tf

```diff
@@ -14,6 +14,37 @@
  * limitations under the License.
  */
 
+variable "cas" {
+  description = "The CAS CAs to add to each environment"
+  type = object({
+    dev = map(object({
```

Comment:
you need more than one CA per env?

Reply:
It might be if multiple services require different cas.